### DISCUSSION TITLE
What is the world's fastest animal on land, air or sea?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#768] [Galaxy Trivia] [answered]
**Bonus:** What speed can it reach?

**Comment by peterbaccaro** [comment, marked as answer]
The world's fastest animal is the **peregrine falcon**, which is a bird of prey that can be found all over the world. The peregrine falcon is the fastest animal on Earth, **capable of reaching speeds of up to 389 kilometers per hour (242 miles per hour)** during its characteristic hunting stoop, which is a high-speed dive towards its prey. This speed allows it to catch other birds in mid-air.

**Comment by SANKARAMDAS** [comment]
The mighty cheetah has been clocked at 75 mph — the speediest runner on the planet. Perhaps you know that the fastest animal in the sea, the sailfish, cruises through the water at 68 mph. In the sky, the Peregrine falcon reigns supreme able to reach a mere 55 mph.
